### PR TITLE
Add Millicast docs

### PR DESCRIPTION
**Pull request by MattiasBuelens**
This adds how-to guides for the new Millicast integrations for both Web and Android.

This also introduces the concept of platform-specific how-to guides for the THEOplayer docs. Previously, all of our guides were shared across all platforms, and we would put all of the platform-specific information into one long guide. With this PR, we now have the ability to have separate guides for separate platforms. As long as they have matching URLs (see #136), users can quickly navigate between them using the platform dropdown.

### CODE REVIEW

File: theoplayer/how-to-guides/android/millicast/getting-started.mdx
Comment (MattiasBuelens) [inline theoplayer/how-to-guides/android/millicast/getting-started.mdx:51]: [This link](https://www.theoplayer.com/docs/theoplayer/v8/api-reference/android/com/theoplayer/android/api/millicast/package-summary.html) is currently dead. We had issues with publishing JavaDoc for the Millicast integration during the last release, and we still haven't figured out how to fix that...

If we can't resolve it, we'll have to remove this link for now.

Comment (GillesMoris-THEO) [reply theoplayer/how-to-guides/android/millicast/getting-started.mdx:51]: Curious 🤔 Let's figure that out, since we're not directing people to this page just yet I'm ok publishing as is while we figure it out and fix it later cause I'd avoid the hassle / risk of forgetting of having to come back and reinstate it but if you prefer that, also good for me.

**Review by GillesMoris-THEO — APPROVED** [review]

File: theoplayer/how-to-guides/android/millicast/getting-started.mdx
Comment (MattiasBuelens) [reply theoplayer/how-to-guides/android/millicast/getting-started.mdx:51]: This should be fixed in the next THEOplayer release. Let's wait for that to land before merging this PR.

**Pull request merged**
